### ISSUE TITLE
Rule change proposal: Enforce `...` over `.split('')` in `prefer-spread`

### ISSUE THREAD

**Issue by fisker** [sindresorhus/eslint-plugin-unicorn#1440]
When splitting string with `''`, spread should be preferred.

```text
'🦄'.split('')
// -> (2) ['\uD83E', '\uDD84']

[...'🦄']
// -> ['🦄']
```

## Fail

```js
foo.split('');
```

## Pass

```js
[...foo];
```

**Comment by sindresorhus** [comment]
This is accepted.
